### PR TITLE
sk machine conf

### PR DESCRIPTION
**Pull request by aniket-l**
- linux-ti-staging[-rt]: Remove patches that do not apply
- u-boot-ti-staging: Remove sk board patches that do not apply now
- Move to using am68-sk/am69-sk yocto machine configs

### CODE REVIEW

**Review by cshilwant — APPROVED** [review]
LGTM!

File: recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend [hunk 1]
```diff
@@ -1,10 +1,5 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```
Comment (cshilwant) [inline recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend:1]: @aniket-l We can remove this line!

**Review by cshilwant — CHANGES_REQUESTED** [review]

**Comment by aniket-l** [comment]
Applied with commit 6b43f754fa2b815068ca2af1c68dc8f884d955ed

File: recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend (hunk 1 above)
Comment (aniket-l) [reply recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend:1]: @cshilwant oops, I merged the PR before seeing the update

Comment (cshilwant) [reply recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend:1]: @aniket-l Ohh, no issues. Can you handle it in a separate PR?

Comment (aniket-l) [reply recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend:1]: @cshilwant Yeah sure will do!

**Pull request closed without merging**